File: leetcode/solutions/1137.N-thTribonacciNumber/tribonacci_test.go
package solution

import "testing"


func Test_1137_NthTribonacciNumber(t *testing.T) {
	result := tribonacci(4)
	expected := 4

	if result != expected {
		t.Errorf("TEST FAILED! got: %#v, want: %#v.", result, expected)
	}

	result = tribonacci(25)
	expected = 1389537

	if result != expected {
		t.Errorf("TEST FAILED! got: %#v, want: %#v.", result, expected)
	}
}

File: leetcode/solutions/1137.N-thTribonacciNumber/tribonacci_test.ts
import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
import tribonacci from "./tribonacci.ts";

Deno.test("1137. N-th Tribonacci Number", () => {
  assertEquals(tribonacci(4), 4);
  assertEquals(tribonacci(25), 1389537);
});
